upgrading/upgrade-roxctl.adoc

```diff
@@ -12,7 +12,8 @@ You can upgrade to the latest version of {rh-rhacs-first} from a supported older
 [IMPORTANT]
 ====
 * You need to perform the manual upgrade procedure only if you used the `roxctl` CLI to install {product-title-short}.
-* For upgrading to {product-title-short} 4.0, you must be using the latest patch release of {product-title-short} 3.74. If you are using an older version, you must first upgrade to {product-title-short} 3.74 before upgrading to {product-title-short} 4.0.
+* For manifest-based (roxctl) Central deployments, you can upgrade from release 3.74 to any latest patch of {product-title-short} version 4.0 through 4.4. However, for full functionality, upgrade to the latest version.
```

Comment:
1st bullet says you can do directly to latest.  2nd bullet recommends you don't.  Should we flip the order on those?

Also a suggestion on the first bullet:

* For manifest-based (roxctl) Central deployments, you can upgrade from release 3.74 to any latest patch of {product-title-short} version 4.0 through 4.4.  It is not possible to upgrade from 3.74 directly to 4.5 or later.  For full functionality Red Hat recommends upgrading central to 4.0 through 4.4 and then to the latest version